perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1128,19 +1129,39 @@ public Member call(ApiCaller ac, Deserializer parms) throws PerunException {
 									parms.readList("allowedStatuses", String.class));
 						} else {
 							// with selected attributes
-							return ac.getMembersManager().getCompleteRichMembers(ac.getSession(),
+							if(parms.contains("allowedGroupStatuses")) {
+								return ac.getMembersManager().getCompleteRichMembers(ac.getSession(),
+									ac.getGroupById(parms.readInt("group")),
+									parms.readList("attrsNames", String.class),
+									parms.readList("allowedStatuses", String.class),
+									parms.readList("allowedGroupStatuses", String.class),
+									parms.readBoolean("lookingInParentGroup"));
+							} else {
+								return ac.getMembersManager().getCompleteRichMembers(ac.getSession(),
```

Comment:
Just a suggestion of a minor readability improvement.

You don't need to duplicate the whole method call and params reading, you can do something like this:
```Java
List<String> allowedGroupStatuses = Collections.emptyList();
if (parms.contains("allowedGroupStatuses")) {
    allowedGroupStatuses = parms.readList("allowedGroupStatuses", String.class);
}
```
This reduces the duplicated code and makes it more clear, how the two calls differ.

Reply:
Code has been simplified.